kolibri/core/assets/src/vue/content-renderer/download-button.vue

```diff
@@ -58,20 +58,16 @@
       downloadMediaText() {
         return this.$tr('downloadMedia');
       },
+      downloadLink() {
+        const filePath = this.availableFiles[0].storage_url;
```

Comment:
Right now, we don't handle there being more than one format, I guess, right? How about supplementary vs non-supplementary? We won't want to hardcode the first element here longer-term.